plugins/Sitemaps/class.sitemaps.plugin.php

```diff
@@ -183,6 +183,15 @@ public function settingsController_sitemaps_create($sender) {
         $sender->setData('Title', t('Sitemap Settings'));
         $sender->setData('isSitePrivate', $this->isSitePrivate);
         $sender->addSideMenu();
+
+        $configurationModule = new ConfigurationModule($sender);
+        $configurationModule->initialize([
+            'Feature.discussionSiteMaps.Enabled' => [
+                'LabelCode' => t('Discussion Based Sitemaps (BETA)'),
```

Comment:
```suggestion
                'LabelCode' => t('Discussion Based Sitemaps', 'Discussion Based Sitemaps (BETA)'),
```

This way the translation code won't change when this goes out of beta. This might also be a case where concatenation is acceptable because then we wouldn't have to retranslate either.